client/my-sites/migrate/section-migrate.jsx

```diff
@@ -365,28 +365,29 @@ class SectionMigrate extends Component {
 					{ translate( 'There was an error with your import.' ) }
 					<br />
 					{ this.state.errorMessage }
+
+					<p className="migrate__info">
+						{ translate(
+							'{{supportLink}}Contact us{{/supportLink}} so we can' +
+								' figure out exactly' +
+								' what needs adjusting, or try again.',
+							{
+								components: {
+									supportLink: (
+										<a
+											href="https://support.wordpress.com"
```

Comment:
I know this is already the case, but would it not make more sense for "Contact us" to link to the contact page (ie. `/help/contact`) rather than a collection of support documents?

Reply:
That's an extremely good point. Thanks for pointing that out! Updated!